app/lib/features/news/pages/add_news_page.dart

```diff
@@ -270,36 +302,22 @@ class _AddNewsState extends ConsumerState<AddNewsPage> {
   }
 
   Widget slideTextPostUI(BuildContext context) {
-    textController.text = selectedNewsPost!.text ?? '';
+    final backgroundColor = ref.watch(
+      newsStateProvider.select((i) => i.currentNewsSlide?.backgroundColor),
+    );
     return Container(
-      padding: const EdgeInsets.symmetric(horizontal: 5.0),
+      padding: const EdgeInsets.all(5.0),
       alignment: Alignment.center,
```

Comment:
Even tough this has a centered alignment, the editor shows up in the top-left :(

Reply:
This is probably due to `shrinkWrap` param of `AppFlowyEditor` not working and occupying entire height.

Reply:
@kumarpalsinh25 might be related to the fact that it isn't the most outer widget but there are these other overlay widgets around we have to figure out...